pkgs/tools/X11/gobble/default.nix

```
@@ -0,0 +1,37 @@
+{ fetchFromGitHub
+, installShellFiles
+, lib
+, pandoc
+, rustPlatform
+, xorg
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gobble";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "EmperorPenguin18";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-g4154Axvjp9jbE0lvMeNGM+v2UxkAsZqt9kPv5bhVK8=";
```

This is preferred now AFAIK, but not a blocking concern.
```suggestion
    hash = "sha256-g4154Axvjp9jbE0lvMeNGM+v2UxkAsZqt9kPv5bhVK8=";
```

---

how about `cargoHash` vs. `cargoSha256`?

---

> how about `cargoHash` vs. `cargoSha256`?

cargoHash is preferred AFAIK

---

Thanks, just fixed that up for #250939 as well.